tests/bazel_run_tests.sh

```diff
@@ -339,10 +339,6 @@ function test_gradle_metadata_is_resolved_correctly_for_jvm_artifact {
   expect_log "@regression_testing_gradle//:com_squareup_okio_okio_jvm"
 }
 
-function test_gradle_versions_catalog {
```

Comment:
Was this deletion intentional?

Reply:
I think that some of these changes were brought in accidentally when rebasing/squashing. I will review.